docs/JLBP-15.md

```diff
@@ -29,35 +27,28 @@ no consistency problem in that case.
 
 See the details specific to each build system in the following sections.
 
-## Libraries using Maven
+## Libraries built with Maven
 
 Create a module containing only the BOM `pom.xml` file and add the BOM module
 to the parent `<modules>` list.
-
 A BOM can inherit from a parent as long as the parent is not part of the build
-  path.
-
-  - Unlike the module POMs of a Maven project, the BOM does not inherit from the
-    parent POM that's used for building other modules of the library.
-
-  - The reason is that a parent will have direct (and possibly transitive)
-    dependencies in its `<dependencyManagement>` section to ensure that its
-    build is consistent, but these dependency versions shouldn't be imported by
-    consumers who import the BOM.
+path. Unlike the module POMs of a Maven project, the BOM does not inherit from the
+parent POM that's used for building other modules of the library.
+The reason is that a parent will have direct (and possibly transitive)
+dependencies in its `<dependencyManagement>` section to ensure that its
+build is consistent, but these dependency versions shouldn't be imported by
+consumers who import the BOM.
 
 Example BOM: [google-cloud-bom](https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/google-cloud-bom/pom.xml).
 
-## Libraries using Gradle
+## Libraries built with Gradle
 
-A Gradle project can either maintain a pom.xml and release it using a
-specially configured module or generate the `pom.xml` file.
+A Gradle project can either generate the `pom.xml` file
+or maintain a pom.xml and release it using a specially configured module.
+gax-java is an example of maintaining a `pom.xml` file:
 
-Example of maintaining a `pom.xml` file: gax-java -
   [build.gradle](https://github.com/googleapis/gax-java/blob/master/gax-bom/build.gradle)
   and
   [pom.xml](https://github.com/googleapis/gax-java/blob/master/gax-bom/pom.xml).
 
-There is not yet an example of a project that generates a `pom.xml` file.
-
-  - Note: This approach has a limitation that the BOM cannot include any
-    classifiers in the dependency specifications.
+This approach has a limitation that the BOM cannot include any classifiers in the dependency specifications.
```

Comment:
The limitation stated here applies to the approach of generating a `pom.xml` file, so since you removed the statement that there is no example, this limitation now seems to apply to the approach of using an explicit `pom.xml` file, which is incorrect.

Reply:
Hmm, I didn't realize that. Why can't generated pom.xml include classifiers?

Reply:
It's a flaw in the gradle support for pom generation.

Reply:
Do you have a link to that feature we can include here? Is it a standard part of gradle or an extra plugin?